src/control-manager.cpp

```diff
@@ -256,7 +274,7 @@ namespace dynamicgraph
             {
               m_emergency_stop_triggered = true;
               SEND_MSG("Estimated torque "+toString(tau(i))+" > max torque "+toString(tau_max(i))+
-                       " for joint "+JointUtil::get_name_from_id(i), MSG_TYPE_ERROR);
+                       " for joint "+ m_robot->model().getJointName(i), MSG_TYPE_ERROR);
```

Comment:
I'd love to avoid this mapping, but to do so we should have the same joint order in the SoT and in the URDF. Is that possible for you?